.buildscript/update_changelog.swift

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/swift 
+
+import Foundation
+
+let contents = try! String(contentsOfFile: "CHANGELOG.md", encoding: .utf8)
+
+var comps = contents.components(separatedBy: "\n")
```

Comment:
Nit: Please use the full word, it's more readable.

Reply:
Will fix. It's just lines, will call them lines.